packaging/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# This Dockerfile is based on the recommendations provided in the
+# Centos official repository (https://hub.docker.com/_/centos/).
+# It enables systemd to be operational.
+FROM centos:7
```

Comment:
This should now also use the version that is used in integration tests

Reply:
Done